packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -169,9 +184,9 @@ export class FocusZone extends React.Component<FocusZoneProps> implements IFocus
     }
 
     if (!this._isInnerZone) {
```

Comment:
probably should move the windowElement null check to 186 and prevent registration.

Reply:
Another thought; if `register` returned the count of active outZone windows, you could avoid having a "getOutZone" method altogether I think.

Reply:
Done